lib/projects/index.ts

```diff
@@ -312,14 +312,6 @@ export async function ensureProjectExists(
         return { projectExists: false };
       }
     }
-    if (
-      isSpecifiedError(err, {
-        statusCode: 401,
-      })
-    ) {
-      logger.error(err.message);
-      process.exit(EXIT_CODES.ERROR);
-    }
```

Comment:
This was redundant, we are calling `logError` right below and that is the pattern we want to use instead of logging the whole `message` prop